sonaranalyzer-dotnet/src/SonarAnalyzer.CSharp/ControlFlowGraph/UcfgInstructionFactory.cs

```diff
@@ -450,6 +450,14 @@ private IEnumerable<Instruction> ProcessBaseMethodDeclaration(BaseMethodDeclarat
         private IEnumerable<Instruction> ProcessMemberAccessExpression(MemberAccessExpressionSyntax memberAccessExpression)
         {
             var memberAccessSymbol = GetSymbol(memberAccessExpression);
+            if (memberAccessSymbol is INamespaceSymbol)
+            {
+                // Every access of a namespace, type or method inside a namespace is represented
+                // by a MemberAccessExpressionSyntax
+                // e.g. Ns1.Ns2.Class1.Class2.Method1(); // has four MemberAccessExpressionSyntax nodes
```

Comment:
The code previously assumed that every _MemberAccessExpressionSyntax_ node related to an invocation, which isn't the case.